main_loop_rate: 30 # [Hz]

# formation params
d: 0.8  # [m]  leader-follower distance in straight line
angle: 80.0 # [deg] leader-follower angle

# flight altitudes
initial_altitude: 1.5
trajectory_altitude: 1.68

# leader to object distance
object_distance: 1.0

# distance between trajectory points
krok_prosta: 0.05  # [m] step on the straight section
krok_luk: 2  # [deg] step on the arch


# precision of positioning at the target point:
tolerance:  
  x: 0.09
  y: 0.09
  z: 0.07
  yaw: 1.3

# 1 - trajectory around object, 2 - trajectory point by point from file
trajectory_mode: 2

# trajectory file path:
csv_path: "/home/bartlomiej/bebop_ws/src/formation_controller/trajectories/testlab.csv"


# collision avoidance:
Ra: 0.65  # alarm zone radius [m]
Rz: 0.45  # forbidden zone radius [m]


